scripts/generate-install-crds-yaml.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+INSTALL_CRDS_YAML="${INSTALL_CRDS_YAML:-hack/install-crds-latest.yaml}"
+
+kubebuilder create config --crds --output "${INSTALL_CRDS_YAML}"
```

Comment:
Testing the presence of `kubebuilder` and exiting with an appropriate message if not found might be useful.

Reply:
Shell script will check the existence of `kuebbuilder`, also how about add the checking in the script which call this script? We can check the return value is 0 or not to decide if this script succeeds or not. Comments?

Same issue also exist for this script https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/generate-install-yaml.sh#L24 , seems we may need some checking for the return value of the script as well.

Reply:
Thanks @pmorie , +1 for this.

@irfanurrehman @pmorie any more comments for other part of this script? Can we get this merged?